### PR TITLE
Minor script changes and annotations fix

### PR DESCRIPTION
**Pull request by rapour**
Some readability improvements to script is suggested. 

The ingress deprecated annotation removed. More can be found [here](https://kubernetes.io/docs/reference/labels-annotations-taints/#kubernetes-io-ingress-class-deprecated).

### CODE REVIEW

File: cicd/scripts/install.sh [hunk 1]
```diff
@@ -1,15 +1,37 @@
...
+# context MUST be set to connect to the k8s cluster
+if [ -z "${CONTEXT}" ]
+then 
+  echo 1>&2 CONTEXT is undefined
+  exit 2
+fi
+
+# environment MUST be either dta or prd 
+shopt -s extglob
+if [[ $ENVIRONMENT == @(dta|prd) ]]; then
```
Comment (mojtabaimani) [inline cicd/scripts/install.sh:18]: env is not limited to dta/prd. it can be anything that user likes to have

File: cicd/scripts/uninstall.sh [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+# context MUST be set to connect to the k8s cluster
+if [ -z "${CONTEXT}" ]
+then 
+  echo 1>&2 CONTEXT is undefined
+  exit 2
+fi
+
+# environment MUST be either dta or prd 
+shopt -s extglob
+if [[ $ENVIRONMENT == @(dta|prd) ]]; then
```
Comment (mojtabaimani) [inline cicd/scripts/uninstall.sh:17]: env is not limited to dta/prd. it can be anything that user likes to have. please remove the condition

**Review by mojtabaimani — CHANGES_REQUESTED** [review]
good work. please just remove the condition for environment.

File: cicd/scripts/install.sh (hunk 1 above)
Comment (rapour) [reply cicd/scripts/install.sh]: Done

File: cicd/scripts/uninstall.sh (hunk 2 above)
Comment (rapour) [reply cicd/scripts/uninstall.sh]: Done

**Review by mojtabaimani — APPROVED** [review]

**Pull request merged**